elastic/tests/test_unit.py

```diff
@@ -145,3 +145,33 @@ def test_get_template_metrics_raise_exception(aggregator, instance):
 def test_get_value_from_path():
     value = get_value_from_path({"5": {"b": [0, 1, 2, {"a": ["foo"]}]}}, "5.b.3.a.0")
     assert value == "foo"
+
+
+def test__get_data_throws_authentication_error(instance):
+    with mock.patch(
+        'requests.get',
+        return_value=MockResponse(status_code=400),
+    ):
+        check = ESCheck('elastic', {}, instances=[instance])
+
+        with pytest.raises(AuthenticationError):
+            check._get_data(url='test.com')
+
+
+@mock.patch.dict('os.environ', {'DDEV_SKIP_GENERIC_TAGS_CHECK': 'true'})
```

Comment:
Why do you need it? It's already defined in the test environment here: 

https://github.com/DataDog/integrations-core/blob/f39318521c94748b2d78329fe5b9e23ba624d905/elastic/hatch.toml#L15-L16

Reply:
Interesting, the test failed in my mac without mocking this env var. That's why I had to mock it here.

Reply:
How did you ran the tests? In your IDE or using hatch?

Reply:
Via command line `ddev test elastic`

Reply:
Works fine on my end without the patch 

```
tests/test_unit.py::test__get_data_creates_critical_service_alert PASSED                                                                                                                                                                                                                                                                                         [100%]
```

🤔 Can you try again? We might have a bug somewhere